wpilibcExamples/src/main/cpp/examples/SwerveBot/cpp/SwerveModule.cpp

```diff
@@ -49,9 +49,13 @@ frc::SwerveModulePosition SwerveModule::GetPosition() const {
 void SwerveModule::SetDesiredState(
     const frc::SwerveModuleState& referenceState) {
   // Optimize the reference state to avoid spinning further than 90 degrees
-  const auto state = frc::SwerveModuleState::Optimize(
+  auto state = frc::SwerveModuleState::Optimize(
       referenceState, units::radian_t{m_turningEncoder.GetDistance()});
```

Comment:
Add `frc::Rotation2d encoderRotation{units::radian_t{m_turningEncoder.GetDistance()}};` and use it in both places.